.work/2026-03-23-code-review-113000.md

```
@@ -0,0 +1,144 @@
+- `src/index.cts`
+- `integration-tests/docker-compose.yml`
+- `integration-tests/tasks.toml`
+- `integration-tests/tests/json-containment.test.ts`
+- `integration-tests/tests/json.test.ts`
+- `integration-tests/tests/keyset.test.ts`
+- `mise.toml`
+- `.cargo/config.toml`
+- `Cargo.lock` (diff stats only)
+
+**Commits reviewed (11):**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)